cmd/controller/main.go

```diff
@@ -30,14 +30,21 @@ func main() {
 	}
 
 	metadataClient := metadata.NewClient(metadata.WithInstrumentation(m.Registry()))
+	var location string
 
-	server, err := app.GetServer(logger, hcloudClient, metadataClient)
-	if err != nil {
-		level.Error(logger).Log(
-			"msg", "failed to fetch server",
-			"err", err,
-		)
-		os.Exit(1)
+	if s := os.Getenv("HCLOUD_SERVER_OVERRIDE_LOCATION_NAME"); s != "" {
```

Comment:
What do you think about calling this var `HCLOUD_VOLUME_DEFAULT_LOCATION` instead?

While this technically provides an override to the server location check, the returned value is used as the default location for volumes which have no location specified.